--- a/librespot-0.6.0/Cargo.toml
+++ b/librespot-0.6.0/Cargo.toml
@@ -68,13 +68,6 @@ url = "2.2"
 
 [features]
 alsa-backend = ["librespot-playback/alsa-backend"]
-portaudio-backend = ["librespot-playback/portaudio-backend"]
-pulseaudio-backend = ["librespot-playback/pulseaudio-backend"]
-jackaudio-backend = ["librespot-playback/jackaudio-backend"]
-rodio-backend = ["librespot-playback/rodio-backend"]
-rodiojack-backend = ["librespot-playback/rodiojack-backend"]
-sdl-backend = ["librespot-playback/sdl-backend"]
-gstreamer-backend = ["librespot-playback/gstreamer-backend"]
 
 with-avahi = ["librespot-discovery/with-avahi"]
 with-dns-sd = ["librespot-discovery/with-dns-sd"]
@@ -82,7 +75,7 @@ with-libmdns = ["librespot-discovery/wit
 
 passthrough-decoder = ["librespot-playback/passthrough-decoder"]
 
-default = ["rodio-backend", "with-libmdns"]
+default = ["alsa-backend", "with-avahi"]
 
 [package.metadata.deb]
 maintainer = "librespot-org"
@@ -104,3 +97,14 @@ assets = [
 
 [workspace.package]
 rust-version = "1.74"
+
+[patch.crates-io]
+ring = { git = "https://github.com/briansmith/ring/" }
+
+[profile.release]
+lto = "thin"
+debug = 0
+panic = "abort"
+incremental = false
+codegen-units = 1
+opt-level = "s"
